prov/efa/src/rxr/rxr_ep.c

```diff
@@ -1896,7 +1896,7 @@ static inline void rxr_ep_check_peer_backoff_timer(struct rxr_ep *ep)
  * @param[in]	ep		end point
  * @param[in]	cqe_to_process	max number of cq entry to poll and process
  */
-static inline void rdm_ep_poll_ibv_cq(struct rxr_ep *ep,
+unit_static inline void rdm_ep_poll_ibv_cq(struct rxr_ep *ep,
```

Comment:
if we are not testing `rdm_ep_poll_ibv_cq` then we do not need this change.

Reply:
I can remove this change